### ISSUE TITLE
Uninstalling avast antivirus with NVDA!

### ISSUE THREAD

**Issue by ghost** [nvaccess/nvda#8037]
When trying to uninstall avast antivirus, with NVDA using the avast uninstall utility, it asks you if you want to uninstall avast with out going into safe mode, whitch is not recommended, and when I chews the button that says no, I get a message saying, you must disable the self protection module in avast antivirus, under settings in the troubleshooting tab, now when I do it with jaws, I chose the button that says no, and avast uninstalls, and asks me to reboot my machine!

**Comment by ehollig** [comment]
@BEdenhofer, is this still the case?

**Comment by fernando-jose-silva** [comment]
I do not know if that's what the reporter of this problem was talking about. However when the avaste is installed, and I go in the windows control panel and I choose to remove the avaste, a window is opened where when walking with tab and nvda can not announce anything relevant.
Nvda announces only button.
When using jaws the screen reader reads the available options, in this case, repair, uninstall, etc.
When pressing space is shown a window that neither the nvda and jaws read, however the focus is on the confirm button, simply pressing another space.
Now in the next window if you walk with tabs the jaws will read what the uninstaller is doing, such as: stopping service, deleting service, deleting files.
Finally, jaws can read the last screen to restart the computer.
When trying to use nvda also in the configuration windows of the avaste it is not possible to read nor a relevant information, the nvda reads only the type of the controls without mentioning what they do.
Jaws can already read the functions of the control panel of the avaste.
This is a problem for those using avaste as a virus scan.
However it is a problem also for those who do not use the avaste, because this avaste offered as extras during the installation of other software, and we can not always unmark it during installation.

**Comment by BEdenhofer** [comment]
It happen's either way, in both cases. No matter how you look at it.

**Comment by jawhien** [comment]
Hi. It seems many antivirus solutions to the problem of accessability, which makes me sad. Or am I mistaken?

**Comment by Adriani90** [comment]
@leonardder, Actually, it is related to Avast which i9s highly inaccessible for NVDA. This is covered in many issues on Github. The whole interface is not accessible. NVDA reports "unknown" when navigating through the interface. jaws does not habe any problems with its interface, it even works in quick navigation mode which is similar to browse mode in NVDA. So I propose creating an app module for Avast since there are lots of people who use Avast simply because it is free.

**Comment by BEdenhofer** [comment]
#We need to contact avast regarding this matter, and explain to them as to what's going on! let me look around for their technical support number, and I'll get back to you's.

**Comment by ehollig** [comment]
Closing as abandoned. Please contact Avast regarding this matter.
